test/sql/updates/setup.continuous_aggs.v2.sql

```diff
@@ -263,3 +263,49 @@ BEGIN
 END $$;
 
 REFRESH MATERIALIZED VIEW mat_ignoreinval;
+
+-- test proper invalidations after update
```

Comment:
Could you modify the comment here? This test case is for inserts after a REFRESH. (We have been using the term invalidations to refer to changes to the raw data that would invalidate a previously computed materialization). These inserts (lines 309-311)  are not logged into the  invalidation_log catalog tables.

Reply:
Right, it's confusing as the terminology has slightly changed in 2.0 (we now also have invalidations for regions that have never been refreshed).  I'm going with `test new data beyond the invalidation threshold is properly handled` here, let me know if that works.